src/main/java/net/tiagofar78/prisonescape/game/prisonbuilding/PrisonBuilding.java

```diff
@@ -21,9 +22,20 @@ public class PrisonBuilding {
 	
 	private List<Chest> _chests;
 	private List<PrisonEscapeLocation> _metalDetectorsLocations;
-	
-	public PrisonBuilding(PrisonEscapeLocation reference) {
+
+	public PrisonBuilding (PrisonEscapeLocation reference) {
 		ConfigManager config = ConfigManager.getInstance();
+		List<ItemProbability> itemsProbabilities = new ArrayList<>();
```

Comment:
add a line before this one and delete the one after. Keep related elements together and unrelated separated.